core/src/main/scala/org/apache/spark/CacheManager.scala

```diff
@@ -126,67 +136,4 @@ private[spark] class CacheManager(blockManager: BlockManager) extends Logging {
       }
     }
   }
-
-  /**
-   * Cache the values of a partition, keeping track of any updates in the storage statuses of
-   * other blocks along the way.
-   *
-   * The effective storage level refers to the level that actually specifies BlockManager put
-   * behavior, not the level originally specified by the user. This is mainly for forcing a
-   * MEMORY_AND_DISK partition to disk if there is not enough room to unroll the partition,
-   * while preserving the the original semantics of the RDD as specified by the application.
-   */
-  private def putInBlockManager[T](
```

Comment:
The idea behind deleting this block of code is the fact that `MemoryStore.putIterator` seems to implement essentially the same logic, using `unrollSafely` and falling back to disk when necessary.